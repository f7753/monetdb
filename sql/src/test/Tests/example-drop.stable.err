stderr of test 'example-drop` in directory 'src/test` itself:


# 15:50:22 >  
# 15:50:22 >  Mtimeout -timeout 480 Mserver -monetrc /var/tmp/manegold.GNU_32-d.6756/monet/.GNU_32-d/Linux2.2.14-12smp/share/Monet/monet.Mtest.conf -db Tests_src_test /var/tmp/manegold.GNU_32-d.6756/sql/src/parser/catalog.mil /var/tmp/manegold.GNU_32-d.6756/sql/src/parser/aserver.mil
# 15:50:22 >  

server listening on port 45123

# 15:50:31 >  
# 15:50:31 >  Mtimeout -timeout 210 sql_aclient  < example-drop.sql
# 15:50:31 >  


# 15:50:31 >  
# 15:50:31 >  Done.
# 15:50:31 >  

