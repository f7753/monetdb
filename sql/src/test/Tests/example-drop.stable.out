stdout of test 'example-drop` in directory 'src/test` itself:


# 20:07:28 >  
# 20:07:28 >  Mtimeout -timeout 180 Mserver "--config=/ufs/niels/data/Linux-i686/etc/monet.conf" --debug=0 --set "monet_mod_path=/ufs/niels/data/Linux-i686/lib/MonetDB" --dbfarm "/ufs/niels/data/Linux-i686/var/dbfarm" --set monet_prompt= --trace "--dbname=mTests_src_test" --set sql_port=50445 "/ufs/niels/data/Linux-i686/share/MonetDB/sqlserver.mil" ; echo ; echo Over..
# 20:07:28 >  

# Monet Database Server V4.3.9
# Copyright (c) 1993-2003, CWI. All rights reserved.
# compiled for i686-pc-linux-gnu/32bit; dynamically linked.
printf("\nReady.\n");

Ready.
quit();

Over..

# 20:07:28 >  
# 20:07:28 >  Mtimeout -timeout 60 Msql -u monetdb -P monetdb --host=stem --port=50445 < example-drop.sql
# 20:07:28 >  

SQL  connected to database mTests_src_test using schema sys
#     1: 0 
#     2: 1 drop view example_view;
#     3: 1 drop table example;

# 19:16:07 >  
# 19:16:07 >  Done.
# 19:16:07 >  

